src/systemctl/systemctl.c

```diff
@@ -161,6 +161,8 @@ static int systemctl_help(void) {
                "                                      otherwise start or restart\n"
                "  try-reload-or-restart UNIT...       If active, reload one or more units,\n"
                "                                      if supported, otherwise restart\n"
+               "  toggle UNIT...                      Start a unit if it's active (running) and\n"
```

Comment:
Opposite? Start if inactive, stop if active

Reply:
Oops, yes it's the opposite. I'll fix it.